encoding/thrift/doc.go

```diff
@@ -109,4 +109,13 @@
 //
 // 	var h handler
 // 	yarpc.InjectClients(dispatcher, &h)
+//
+// Automatically Sanitizing TChannel Contexts
+//
+// Contexts created with `tchannel.ContextWithHeaders` are incompatible with yarpc clients generated from thrift.
+// Using such a context will cause a yarpc client to error on any call. Using the `sanitize-tchannel` flag will
+// generate a yarpc client such that all tchannel headers from any context supplied are removed before making a yarpc call.
+// the option can be used like so:
+//
+// 	thriftrw --plugin yarpc --sanitize-tchannel myservice.thrift
```

Comment:
This won't work. For the flag to be passed to the plugin, it should be passed as part of the --plugin argument.

    --plugin "yarpc --sanitize-tchannel"